modules/claims_api/spec/sidekiq/evidence_waiver_builder_job_spec.rb

```diff
@@ -44,4 +44,16 @@
       end
     end
   end
+
+  context 'when the claims_api_ews_uploads_bd_refactor BD refactor feature flag is enabled' do
```

Comment:
Please add `context 'when claims_api_ews_uploads_bd_refactor is disabled'`

Reply:
Context added for feature disabled.